modules/parquet/src/parquetjs/compression.ts

```diff
@@ -86,17 +86,17 @@ export function decompress(method: ParquetCompression, value: Buffer, size: numb
   return toBuffer(compressedArrayBuffer);
 }
 
-/**
+/*
  * Inflate a value using compression method `method`
- *
-export function inflate(method: ParquetCompression, value: Buffer, size: number): Buffer {
+ */
+export function inflate(method: ParquetCompression, value: Buffer): Buffer {
   if (!(method in PARQUET_COMPRESSION_METHODS)) {
     throw new Error(`invalid compression method: ${method}`);
   }
-
-  return PARQUET_COMPRESSION_METHODS[method].inflate(value, size);
+  // @ts-ignore
```

Comment:
Why did we remove the `size` addition? I recently added it since it seemed that it was used by one of the compressions.

Reply:
Returned back.